docs-source/content/userguide/managing-domains/model-in-image/runtime-updates.md

```diff
@@ -153,9 +151,9 @@ As was mentioned in the [overview](#overview), one way to tell the operator to a
    fi
    ```
 
-#### Using the WDT Discover Domain Tool
+#### Using the WDT Discover Domain and Compare Model Tools
 
-The WebLogic Deploy Tooling [Discover Domain Tool](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/discover.md) generates model files from an existing domain home. You can use this tool to help determine the model file contents you would need to supply to update an existing model.
+The WebLogic Deploy Tooling [Discover Domain Tool](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/discover.md) generates model files from an existing domain home, and the WebLogic Deploy Tooling [Compare Model Tool](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/compare.md) compares two domain models and generates the YAML for updating the first domain to the second domain. You can use these tools in combination to help determine the model file contents you would need to supply to update an existing model.
```

Comment:
and the WebLogic Deploy Tooling [Compare Model Tool] > and the [Compare Model Tool]
YAML -> YAML file

Reply:
I don't see this corrected.

Reply:
oops - sorry - missed that one

Reply:
it's corrected now